src/app/view/components/dot-navigation/services/dot-navigation.service.spec.ts

```diff
@@ -209,9 +209,17 @@ describe('DotNavigationService', () => {
                 }
                 counter++;
             });
+            service.collapseMenu();
+        });
+    });
 
+    describe('collapseMenu', () => {
+        it('should collapse menu and call closeAllSections', () => {
+            expect(service.collapsed$.getValue()).toBe(true);
+            spyOn(service, 'closeAllSections');
```

Comment:
We can't spy on a method of the service we're testing, we need to test what happen when that method gets called.

Reply:
done